### PR TITLE
190521 - BYTE3 Pull Request

### PR DESCRIPTION
**Pull request by tomsalter1999**
When submitting a new or existing set details change, make sure you have followed the following steps:

### Adding New Set Details
- [x]  Add a new file (or update existing file) into the `set-details` folder that is named as `[insert-your-set-address].setdetails.json` (i.e. 0x1494CA1F11D487c2bBe4543E90080AeBa4BA3C2b.setdetails.json). Make sure to provide the [checksum address](https://ethsum.netlify.app/) for your Set
- [x] Verify the JSON follows the correct schema according to this [template](https://github.com/SetProtocol/uniswap-tokenlist/blob/main/set-details.schema.json). You can verify it by pasting the schema into this [schema validator](https://www.jsonschemavalidator.net) and checking if your input JSON matches properly. If you need to have multi-line JSON, you can use a HJSON converter to help with writing it and then paste in the final valid JSON spit out from it (see README for example)
- [x] If adding brand new token, make sure to add it to the set.tokenlist.json as well following the correct schema from Uniswap: https://github.com/Uniswap/token-lists/blob/master/src/tokenlist.schema.json

### Updating Existing Set Details
- [x] If updating existing set details, update the version number appropriately (see the JSON schema above for version numbering explanation).
- [x] When generating a signature, make sure the version signed is for the one reflected in the version details for your entry.

### Signature (Required)

- [x] Attach the proper signature. You can generate one by going to the page for your Set on TokenSets and signing in. Tap on your Set's name and follow the instructions provided (i.e. 0x21fbf0696d5e0aa2ef41a2b4ffb623bcaf070461d61cf7251c74161f82fec3a4370854bc0a34b3ab487c1bc021cd318c734c51ae29374f2beb0e6f2dd49b4bf41c):

0x6e09f9ff6d8ea84673d317b401815c722f5c81d7af210b8f5543a3b44f3f1e7e5f7604a9489c252ef21b78f75b31a96f96b0b6717108867405a3044b8a4c96e91b

### CODE REVIEW

File: set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "Bitcoin Network Demand Set",
+    "setSymbol": "BYTE3",
+    "setDecimals": 18,
+    "setIcon": "",              
+    "setDescription": "**The BYTE set is not a regulated investment vehicle. The set follows the Bitcoin Network Demand Model. Rebalances are executed through the SetsProtocol** /n/n**An investor may get back less than the amount invested. Information on past performance, where given, is not a guide to future performance** /n/nThe algorithm behind the BYTE3 set incorporates four network demand metrics across six different indicators, to give an overall allocation score. These metrics are Network Fees, Transaction Value, Velocity and Miners’ Change in Inventory. The Litepaper for the Bitcoin Network Demand Model is available here: https://bytetree.com/assets/uploads/bitcoin-network-demand-model-byte3.pdf /n/n**Please note that we reserve the right to amend the operation of the set**"
```
Comment (asoong) [inline set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json:8]: Comma required after this

File: set-details/Test File [hunk 2]
```diff
@@ -0,0 +1 @@
+[0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99].setdetails.json
```
Comment (asoong) [inline set-details/Test File:1]: Remove this file!

**Review by asoong — CHANGES_REQUESTED** [review]

File: set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "Bitcoin Network Demand Set",
+    "setSymbol": "BYTE3",
+    "setDecimals": 18,
+    "setIcon": "",              
+    "setDescription": "**The BYTE set is not a regulated investment vehicle. The set follows the Bitcoin Network Demand Model. Rebalances are executed through the SetsProtocol** /n/n**An investor may get back less than the amount invested. Information on past performance, where given, is not a guide to future performance** /n/nThe algorithm behind the BYTE3 set incorporates four network demand metrics across six different indicators, to give an overall allocation score. These metrics are Network Fees, Transaction Value, Velocity and Miners’ Change in Inventory. The Litepaper for the Bitcoin Network Demand Model is available here: http://bit.ly/ndmb3 /n/n**Please note that we reserve the right to amend the operation of the set**",
+    "managerName": "ByteTree",
+    "managerIcon": "",
```
Comment (asoong) [inline set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json:10]: Icon links need to be filled in!

File: set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "setDetails": {
+    "chainId": 1,
+    "setName": "Bitcoin Network Demand Set",
+    "setSymbol": "BYTE3",
+    "setDecimals": 18,
+    "setIcon": "",
```
Comment (asoong) [inline set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json:7]: Icon links need to be filled in!

**Review by asoong — CHANGES_REQUESTED** [review]
2 things:
* Make sure you add the correct asset and manager logo paths
* Add your entry to the set uniswap token list as well so it will appear on Uniswap if you want later. See example below.

https://github.com/SetProtocol/uniswap-tokenlist/pull/19/files

File: set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json (hunk 4 above)
Comment (tomsalter1999) [reply set-details/0xFF32b6dc663e9AA3Ef61117749C6cCC168100b99.setdetails.json]: Done

**Pull request merged**